tests/modules/individuals/resources/test_individual_names.py

```diff
@@ -26,137 +28,340 @@ def test_get_set_individual_names(db, flask_app_client, researcher_1, request, t
 
     assert len(sighting.encounters) > 0
     enc = sighting.encounters[0]
+    bad_guid = '00000000-0000-0000-2170-000000000000'
+
+    # first we try with bunk data for one name
+    individual_data_in = {
+        'names': [
+            {'context': 'A', 'value': 'value-A'},
+            {'conxxxx': 'B', 'value': 'value-B'},
+        ],
+        'encounters': [{'id': str(enc.guid)}],
+    }
+    individual_response = individual_utils.create_individual(
+        flask_app_client, researcher_1, 400, individual_data_in
+    )
 
-    try:
-
-        individual_data_in = {
-            'names': {
-                'defaultName': 'Godzilla',
-                'nickname': 'Doctor Atomic',
-                'oldName': 'critter-271',
-            },
-            'encounters': [{'id': str(enc.guid)}],
-        }
-
-        individual_response = individual_utils.create_individual(
-            flask_app_client, researcher_1, 200, individual_data_in
-        )
-
-        assert individual_response.json['result']['id'] is not None
-
-        individual_id = individual_response.json['result']['id']
+    # now with bunk preferring_users user guid, but name fixed
+    individual_data_in = {
+        'names': [
+            {'context': 'A', 'value': 'value-A'},
+            {'context': 'B', 'value': 'value-B', 'preferring_users': [bad_guid]},
+        ],
+        'encounters': [{'id': str(enc.guid)}],
+    }
+    individual_response = individual_utils.create_individual(
+        flask_app_client, researcher_1, 400, individual_data_in
+    )
 
-        individual_json = individual_utils.read_individual(
+    # fix the name and preferring_users and try again (will make our individual)
+    individual_data_in['names'][1] = {
+        'context': 'B',
+        'value': 'value-B',
+        # TODO FIXME  POST + preferring_users is currently now working
```

Comment:
TODO ?  is this planned to be fixed later?

Reply:
i actually want to test this a bit more before this gets merged.  i may have made some schema/parameter changes that could have helped this.  as for future-fix, i will ask ben what the likelihood of needing this for the ui/ux would be.